cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -618,80 +618,89 @@ public ClientBuilder readTimeout(long readTimeout, TimeUnit readTimeoutUnit) {
     }
 
     /**
-     * Sets Cloudant client credentials by inspecting the VCAP_SERVICES Cloud Foundry environment
-     * variable. The VCAP_SERVICES variable contains connection information to access a Bluemix
-     * service instance.
+     * Sets Cloudant client credentials by inspecting a service information JSON string. This object
+     * takes the form of a VCAP_SERVICES environment variable which is a JSON object that contains
+     * information that you can use to interact with a service instance in Bluemix.
      * <P>
-     * Note: Only a single Cloudant service must exist in VCAP_SERVICES. If there are multiple bound
-     * Cloudant service instances then use {@link #bluemix(String)} to select an instance by name.
+     * Note: Only a single Cloudant service instance must exist in the vcapServices JSON. If
+     * multiple bound instances are present then use {@link #bluemix(String, String)} to select a
+     * Cloudant service instance by instance name.
      * </P>
      *
+     * @param vcapServices vcapServices service information JSON string, for example the contents of
+     *                     VCAP_SERVICES environment variable
      * @return a new ClientBuilder for the account
+     *
+     * @throws IllegalArgumentException if the Cloudant service instance information is not
+     * available in the vcapServices JSON
```

Comment:
I think we need to go into a bit more depth here, since this exception can be thrown if there is more than one cloudant service, if there are no cloudant services etc.